src/shell/doom.c

```diff
@@ -24,7 +24,7 @@
  *   Raise the fatal exception immediately if the first task fails,
  *   e.g. calls exit with a nonzero value or is terminated by signal.
  */
-#define FLUX_SHELL_PLUGIN_NAME "doom"
+#define FLUX_SHELL_PLUGIN_NAME "task-exit"
```

Comment:
my anal retentiveness would want to rename the file too ... but that's me.

Reply:
I was going to do that and started replacing the `doom_` prefixed functions with `task_exit_`, and then realized that makes a mess of some function names like

doom_exit_host => task_exit_exit_host
doom_task_exit => task_exit_task_exit

So I said "bah!" and left it as is.

Reply:
Oh it just occurred to me, your commit message says to rename the plugin "exit-timeout" but this is "task-exit".  I think "exit-timeout" is probably better, since it could be a timeout do to the shell and not a task per se.

Reply:
Ah thanks, I'll fix.  The plugin doesn't always employ a timeout, but it's always triggered by task exit (or by proxy - if a shell exits its tasks have exited).  I think task-exit is probably the right name.